### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by samivari**
# Description
Added content to ReadMe File... Needs JSON responses to be complete

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,135 @@
...
+## Development setup
+```ruby 2.7.4```
+```rails 5.2.6```
+## Gems
+![pry v badge](https://img.shields.io/gem/v/pry?color=blue&label=pry)
+![shoulda-matchers v badge](https://img.shields.io/gem/v/shoulda-matchers?label=shoulda-matchers)
+![rspec v badge](https://img.shields.io/gem/v/rspec?color=orange&label=rspec)
+![simplecov v badge](https://img.shields.io/gem/v/simplecov?color=green&label=simplecov)
+![json-apiserializer](https://img.shields.io/badge/json-apiserializer-blue)
+![faraday](https://img.shields.io/badge/faraday-red)
```
Comment (glsorensen) [inline README.md:30]: Nice badges!

**Review by glsorensen — CHANGES_REQUESTED** [review]
Looks good once I get you the Postman Docs, and the schema image is updated

**Review by samivari — COMMENTED** [review]
ReadMe updated

**Review by samivari — COMMENTED** [review]
changes reviewed

**Pull request merged**